### PR TITLE
Fix client configuration getting overwritten when mutliple clients are available

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
Fixes https://github.com/ballerina-platform/ballerina-library/issues/5892
## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests
- [ ] Updated the spec
- [x] Checked native-image compatibility

### CODE REVIEW

File: ballerina-tests/tests/79_multiple_client_configurations_client.bal [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Bhashinee) [inline ballerina-tests/tests/79_multiple_client_configurations_client.bal:1]:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment (Bhashinee) [reply ballerina-tests/tests/79_multiple_client_configurations_client.bal:1]: Check other places as well.

**Review by Bhashinee — APPROVED** [review]

File: ballerina-tests/tests/79_multiple_client_configurations_pb.bal [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (dilanSachi) [inline ballerina-tests/tests/79_multiple_client_configurations_pb.bal:1]:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

File: ballerina-tests/tests/79_multiple_client_configurations_pb.bal [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (dilanSachi) [inline ballerina-tests/tests/79_multiple_client_configurations_pb.bal:3]:
```suggestion
// WSO2 LLC. licenses this file to you under the Apache License,
```

File: ballerina-tests/tests/79_multiple_client_configurations_service.bal [hunk 4]
```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (dilanSachi) [inline ballerina-tests/tests/79_multiple_client_configurations_service.bal:1]:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

File: ballerina-tests/tests/79_multiple_client_configurations_service.bal [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (dilanSachi) [inline ballerina-tests/tests/79_multiple_client_configurations_service.bal:3]:
```suggestion
// WSO2 LLC. licenses this file to you under the Apache License,
```

**Review by Bhashinee — APPROVED** [review]

**Pull request merged**